doc/api/fs.md

```diff
@@ -3072,6 +3072,32 @@ extended part is filled with null bytes (`'\0'`):
 
 If `len` is negative then `0` will be used.
 
+The `fs.read()` method reads data from the file specified by the file
+descriptor (`fd`). The `length` argument indicates
+the maximum number of bytes that Node.js will attempt
+to read from the kernel. However, the actual
+number of bytes read (`bytesRead`) can be lower
+than the specified `length` for various reasons.
+
+For example:
+
+* If the file is shorter than the specified `length`, `bytesRead`
+  will be set to the actual number of bytes read.
+* If the file encounters EOF (End of File) before the buffer could be filled,
+  the callback will be called with an error.
+* If the file is longer than the specified `length`, only `length` bytes
+  will be read from the file, and `bytesRead` will be set to `length`.
```

Comment:
This is *not always* guaranteed and looks redundant.

Reply:
Can I delete this place?

Reply:
Yes, I think so.

Reply:
okey, thank you